toolchain/check/testdata/impl/fail_extend_impl_scope.carbon

```diff
@@ -16,6 +16,14 @@ interface I {}
 // CHECK:STDERR:
 extend impl i32 as I {}
 
+fn F() {
+  // CHECK:STDERR: fail_extend_impl_scope.carbon:[[@LINE+4]]:3: error: `extend impl` can only be used in a class [ExtendImplOutsideClass]
+  // CHECK:STDERR:   extend impl {} as I {}
+  // CHECK:STDERR:   ^~~~~~~~~~~~~~~~~~~~~
+  // CHECK:STDERR:
+  extend impl {} as I {}
+}
+
```

Comment:
You might consider using split files in order to test that each diagnostic occurs independently. Right now, one diagnostic could be removed and the other could stay without affecting overall test pass/failure.

Reply:
Done. Also made it a no_prelude test while I was at it.